app/repository/hbase/enterprise/EnterpriseUnitRowMapper.scala

```diff
@@ -14,22 +15,61 @@ object EnterpriseUnitRowMapper extends RowMapper[Enterprise] {
   override def fromRow(variables: Row): Option[Enterprise] =
     for {
       ern <- variables.fields.get(ern)
-      entref <- variables.fields.get(entref)
+      entrefStr = variables.fields.get(entref)
```

Comment:
Did you mean entrefOpt?
We know that all the raw values are strings - the thing we usually emphasise is the options (as the lack of a generator is somewhat subtle).
Same for tradingStyle etc...